toolchain/binutils/Config.in

```diff
@@ -6,22 +6,6 @@ choice
 	help
 	  Select the version of binutils you wish to use.
 
-	config BINUTILS_USE_VERSION_2_37
-		bool "Binutils 2.37"
-		select BINUTILS_VERSION_2_37
-
-	config BINUTILS_USE_VERSION_2_38
-		bool "Binutils 2.38"
-		select BINUTILS_VERSION_2_38
-
-	config BINUTILS_USE_VERSION_2_39
-		bool "Binutils 2.39"
-		select BINUTILS_VERSION_2_39
-
-	config BINUTILS_USE_VERSION_2_40
```

Comment:
Please drop all older versions than the one currently used by default except the version which was used in the previous release.
Please keep version 2.40, 2.42 and 2.43 and remove version 2.41 instead.

Reply:
I updated the commit.
Thank you for review!